x-pack/plugins/stack_alerts/server/alert_types/index_threshold/README.md

```diff
@@ -123,227 +120,6 @@ server    log   [17:32:10.060] [warning][actions][actions][plugins] \
 [now-iso]: https://github.com/pmuellr/now-iso
 
 
-## http endpoints
+## Data Apis via the TriggersActionsUi plugin and its http endpoints
```

Comment:
ahhh ... now I see, the index action UIs are also using these APIs I guess.  Confused me for a bit as to why the APIs moved there.  I think this makes sense for now, but does seem a little weird - I guess it feels like maybe we need YAAP (yet another alerting plugin) where these back-end APIs might live, to separate it out a bit from the mainly UI plugin (I guess I kinda alluded to that in a previous comment re: the new plugin dependencies).  A change to make later, if we ever need to ...

Reply:
TBH I think these APIs belong in the `data` plugin... hence the name ;)
For now it makes sense in the Triggers Actions UI as it's a plugin we can expect to always be enabled if actions or alerts are in use